ui_testing/pages/header_page.py

```diff
@@ -185,11 +186,16 @@ def click_on_the_confirm_message(self):
         self.base_selenium.click(element='general:confirm_pop')
         self.sleep_small()
 
-    def click_on_clear_all(self):
-        self.base_selenium.LOGGER.info('Press on the overview button')
-        self.base_selenium.click(element='user_management:clear_all')
+    def click_on_overview_btn(self):
+        self.base_selenium.LOGGER.info('Press on the overview button button')
+        self.base_selenium.click(element='user_management:overview_btn')
         self.sleep_small()
 
+    def click_create_new_user(self):
+        self.base_selenium.LOGGER.info('Press on the create new user button')
+        self.base_selenium.click(element='user_management:create_user_button')
+        self.sleep_small()
+        
```

Comment:
why do we have methods for the clicking action? I already asked you before to not do that as much as possible. Just create a method for overviewing action and another method for creat_new_user

Reply:
@0xIslamTaha I changed a lot of methods to do that but some of them need sleep after the click action so why we don't keep on that